dts/arm/st/g4/stm32g474.dtsi

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/g4/stm32g4.dtsi>
```

Comment:
Remove this line

Reply:
line "#include <st/g4/stm32g4.dtsi>" replaced by "#include <st/g4/stm32g431.dtsi>"

Reply:
I still see both lines actually